frontend/src/Frontend/Network.hs

```diff
@@ -404,10 +403,10 @@ defaultTransactionTTL = TTLSeconds (8 * 60 * 60) -- 8 hours
 
 -- Taken from https://github.com/kadena-io/chainweb-node/blob/85688ea0182d1b1ab0d8d784a48b4851a950ec7a/src/Chainweb/Chainweb.hs#L344
 chainwebGasLimit :: Num a => a
-chainwebGasLimit = 1e5
+chainwebGasLimit = 600
```

Comment:
This is still 100,000 (see upstream). This really should be renamed to `chainwebGasLimitMaximum` or something, to set it apart from the user configurable gas limit (which has the default from below)